lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -765,6 +794,15 @@ SessionConfig SessionState::get_config() const {
   return config_;
 }
 
+uint32_t SessionState::get_local_teid() const {
+  return local_teid;
+}
+
+void SessionState::set_local_teid(uint32_t teid) {
+  local_teid = teid;
+  return;
+}
```

Comment:
@gurrapualt  Also do we not need to update the `SessionStateUpdateCriteria` to persist the value?

Reply:
Now changed added, updated criteria as well.

Reply:
I still think we should use `agw_teid`. The reason is protos have `agw_teid` and `enb_teid` so matching it with `agw_teid = agw_teid` reduces possible issues of doing `local_teid = enb_teid`

In case of a bug, finding the issue becomes very difficult